packages/www/pages/docs/guides/debugging-guide.mdx

```diff
@@ -0,0 +1,72 @@
+
+#### Prerequisite: Locate your playback URL. 
+
+Your playback URL is specific to the stream ingest region. 
+- This [Livepeer.com API guide](/docs/guides/api/playback-a-live-stream) touches on finding your playback URL by combining a base URL with a `playbackId`. If you don't know your stream `playbackId`, you can retrieve it via [`GET /stream/<id>`](/docs/api/stream/get-stream).
+- You can also find the playback URL on the Livepeer.com dashboard stream page, `https://livepeer.com/app/stream/<stream-id>`. Learn more [here](/docs/guides/dashboard/playback-a-stream).
+
+## Step 1
+
+Load your playback URL in the [Livepeer.com test player](/app/test-player). If you get a "Stream not found" error, confirm that your URL is correct. It should look something like this:  https://region-cdn.livepeer.com/hls/123456abcdef7890/index.m3u8
+
+## Step 2
+
+What seems to be the problem you are experiencing?
+
+- **The video is not playing in my application at all.** Go to the next step.
+- **The video is playing, but the quality is poor.** Go to step 5.
+- **I don't know.**: Go to step 4.
+
+## Step 3
+
+Does anything playback in the [Livepeer.com test players](/app/test-player)?
+
+- **Yes, the quality is great:** There may be something wrong with your player configuration. Start by checking if your player is loading the correct playback URL.
+- **No:** Go to the next step.
+- **Yes, but the quality is poor:** Go to step 5
+
+## Step 4
+
+Check the stream status below the Livepeer.com test player. Is it active?
+
+- **No:** No video is being ingested. Confirm your or your streamer’s broadcast software RTMP configuration is set up properly. If you are using OBS to stream into Livepeer, check out steps 4-5 in [this blog post](/blog/first-livepeer-stream-in-five-minutes). 
+- **Yes:** Go to the next step.
+
+## Step 5
+
+Check if you are transcoding. Is there only one `profile` listed next to `Playback settings` below the Livepeer.com test player? 
+
+- **Yes:** You are not transcoding your source stream, and only the source profile is available for playback (no ABR) via the Livepeer.com CDN. There may be something wrong with your source stream. If you can, instruct the streamer to set the keyframe interval to 2 in their broadcasting software; this can help.  In general, make sure your broadcasting workflow is stable. The RTMP stream hitting Livepeer's ingest server has problems. Try ensuring a stronger internet connection at the source via a wired ethernet connection or by broadcasting on a more reliable network.
+<br />Or, if the resolution, FPS and/or bitrate of the source stream is very high and if your internet bandwidth is low, playback will be slow and choppy. Consider transcoding your livestreams. If no transcoding renditions are available for playback, you are not making use of ABR.
+- **No:** If two or more `profiles` are listed, go to the next step.
+
+## Step 6
```

Comment:
This step will be confusing for anyone creating a stream on livepeer.com and debugging. Could we add directions on skipping it if stream created on livepeer.com, or something of the like?